### ISSUE TITLE
Missing cover make app cash.

### ISSUE THREAD

**Issue by jojoto** [Monochromish/Last.fm-Discord-Rich-Presence#3]
When an album has no cover on last.fm, the API returns nothing. The value of "data.cover" is set to "Null".
As [discord-rpc](https://github.com/discord/discord-rpc) requires a value, the script crashes.

https://github.com/Monochromish/Last.fm-Discord-Rich-Presence/blob/7ba1ed716d8af3675fa33e87113c439bfd7e5e64/src/index.js#L145

A workaround could be setup before submitting the data to discord :

After line [133 of index.js](https://github.com/Monochromish/Last.fm-Discord-Rich-Presence/blob/7ba1ed716d8af3675fa33e87113c439bfd7e5e64/src/index.js#L133)
``` JS
if (!data.cover){
    data.cover = FallbackImageUrlOrBase64;
}
```

**Comment by Monochromish** [comment]
Hmm, that's weird, could you give me an example of a song or two where you're coming across this issue?
When I play a music that's either entirely not available on Last.fm or has no album cover uploaded to Last.fm, the API provides [this picture](https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png) in place of album cover so it never crashes for me.

![](https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png)

**Comment by jojoto** [comment]
I've also had this image with some of TIDAL's tracks, but this problem often happens with "Jamendo" music.
Take [this track](https://www.jamendo.com/track/1925220/on-repeat) for example.

the api returns to me:
```JSON
"artist": {
  "mbid": "",
  "#text": "Seth Power"
},
"streamable": "0",
"image": [
  {
    "size": "small",
    "#text": ""
  },
  {
    "size": "medium",
    "#text": ""
  },
  {
    "size": "large",
    "#text": ""
  },
  {
    "size": "extralarge",
    "#text": ""
  }
],
"mbid": "",
"album": {
  "mbid": "",
  "#text": ""
},
"name": "On Repeat",
"@attr": {
  "nowplaying": "true"
},
"url": "https://www.last.fm/music/Seth+Power/_/On+Repeat"
```
For the moment, I think a workaround should be envisaged when this rare case occurs to avoid possible crashes of the app.
I'll also have to check the last.fm forums to see if this problem has already been mentioned.

**Comment by Monochromish** [comment]
Okay, I'll try to reproduce the issue and come up with a bug fix shortly, thanks!

**Comment by Monochromish** [comment]
Couldn't try Jamendo Music, although I tried Tidal and seems like I've understood the problem.
Apparently, when certain music is being streamed on platforms such as Jamendo, the Last.fm API fails to provide some information including the album cover art.
To avoid this, I've pushed and created a release for this bug which should *hopefully* provide [this picture](https://lastfm.freetls.fastly.net/i/u/300x300/2a96cbd8b46e442fc41c2b86b821562f.png) when no album cover art is being provided; [check out the release](https://github.com/Monochromish/Last.fm-Discord-Rich-Presence/releases/tag/1.0.5).

I'll have this issue opened at least for a few days in case you come across any more errors, feel free to share them with me here!
